cli/src/commands/theme.ts

```diff
@@ -0,0 +1,283 @@
+/**
+ * /theme command - Switch between different themes
+ */
+
+import type { Command, ArgumentProviderContext } from "./core/types.js"
+import type { CLIConfig } from "../config/types.js"
+import { getThemeById, getAvailableThemes, getThemeType } from "../constants/themes/index.js"
+import { getBuiltinThemeIds } from "../constants/themes/custom.js"
+import { logs } from "../services/logs.js"
+
+// Cache for config to improve performance
+let configCache: { config: CLIConfig | null; timestamp: number } | null = null
+const CONFIG_CACHE_TTL = 5000 // 5 seconds
+
+/**
+ * Get config with caching to improve performance
+ *
+ * Error scenarios handled:
+ * - Config file not found: Uses DEFAULT_CONFIG
+ * - Config file corrupted/unreadable: Falls back to DEFAULT_CONFIG
+ * - Network/permission issues: Falls back to DEFAULT_CONFIG with shorter cache
+ * - Invalid config structure: Relies on loadConfig's built-in validation/defaults
+ *
+ * Follows the same error handling pattern as other config operations in persistence.ts
+ */
+async function getConfigWithCache(): Promise<{ config: CLIConfig }> {
```

Comment:
Unless you feel strongly about the cache, I would vote to remove it. In my experience these types of caches just add complexity.

Also, let's remove the catch behavior and just let the method throw an error when loading the config fails. I can't imagine what type of error would cause the method to fail, but I'm pretty sure that we don't want it to fall back to the default config when it does.